database/youtube/ui-ux.json

```diff
@@ -4,23 +4,23 @@
         "description": "The UI/UX Design Specialization brings a design-centric approach to user interface and user experience design, and offers practical, skill-based instruction centered around a visual communications perspective, rather than on one focused on marketing or programming alone",
         "url": "https://www.youtube.com/watch?v=pyQAiRuqUSM",
         "category": "youtube",
-        "subcategory": "ux-designing",
+        "subcategory": "ui-ux",
```

Comment:
Let's change this to `ui-ux-designing`. Also, please take a look at SideBar component's code to check how we're displaying the title. So the title would match.


<img width="1086" alt="Screenshot 2024-02-05 at 10 31 45 AM" src="https://github.com/rupali-codes/LinksHub/assets/93012310/d6d27e01-2842-4d04-82dd-72f3d4c8912c">

```suggestion
        "subcategory": "ui-ux-designing",
```